README.md

```diff
@@ -32,6 +32,14 @@ Rendering Examples:
 Source code for this example can be found [here](examples/cairo_example.py).
 
 
+Install from source:
+```
+$ git clone https://github.com/curtacircuitos/pcb-tools.git
+$ cd pcb-tools
+$ make

```

Comment:
Ah, I didn't know. Hence the need for install instructions :-)
I'll update the PR.